src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const LONG_TERMIN = 7;
```

Comment:
The constant name `LONG_TERMIN` seems to be a typo. Consider renaming it to `LONG_TERM` to match the checklist and to make it more descriptive and correct.